apps/laboratory/tests/README.md

```diff
@@ -5,9 +5,14 @@ We use Playwright as our functional test runner. It's configured to try multiple
 - Browsers: Chromium/Firefox
 - Modal flavors: default/SIWE/email
 
+## Setup
+
+- Make sure your `.env` is set up (see `.env.example` for reference)
```

Comment:
@arein @svenvoskamp Something I'm currently still unclear on: where can I find the right values for local runs for the following:

```
NEXTAUTH_SECRET=""
NEXTAUTH_URL=""
MAILSAC_API_KEY=""
```

Without them the SIWE and Email tests seem to fail, and I wasn't able to find setup instructions for these values

Reply:
you can get them from Vercel / Labs deployment

Reply:
Mailsax: 1Password `Mailsac Web3Modal API Key`

Reply:
Thanks both, will update instructions accordingly for the next dev doing this 👍

Reply:
@glitch-txs do you know what `NEXTAUTH_URL` should be set to? Not defined in 1password or in w3m lab Vercel env variables and if I leave it blank/default I get `ERR_INVALID_URL`.

Reply:
I don't know but you can leave it as `NEXTAUTH_URL=""` I think

cc @Cali93

Reply:
it's from here: https://next-auth.js.org/configuration/options#nextauth_url
but only refers to prod in docs

Reply:
Ok so I figured out what the confusing thing was:
* `NEXTAUTH_URL` needs to point to the lab itself, so explicitly `http://localhost:3000` works but it's actually simpler to just omit it completely since it doesn't need to be set locally.
* Using this `.env.example` as-is was causing it to pick up the URL as empty string `""`